docs/WebpackDevServerService.md

```diff
@@ -0,0 +1,35 @@
+# WebpackDevServer Service
+WebpackDevServer Service is provided as a convenience to start a webpack-dev-server and return a promise when the webpack compiler is complete. This enables test setup to be complete before testing is began. This service is utilized by the the default nightwatch configuration and must be provided as a service to webdriver.io if the webpack-dev-server is used to host your test pages.
+
+## Options
+
+In the wdio.conf.js you can pass a configuration object with the following structure:
+
+* **webpackConfig** - the webpack configuration used to start the webpack-dev-server. Must be provided to use this service.
+* **webpackPort** - the port to start the webpack-dev-server on. Defaults to port 8080.
+* **webpackDevServerConfig** - the webpack-dev-server configuration to be passed to the webpack-dev-server. Defaults to { quiet: true, hot: false, inline: false }.
+
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const webpackConfig = require('./webpack.config.js');
+const WebpackDevService = require('terra-toolkit/wdio/services/index').WebpackDevService;
+const localIP = require('ip');
+
+const port = 8080;
+
+const config = {
+  ...wdioConf.config,
+
+  // Point base URL at the site to be tested
+  baseUrl: `http://${localIP.address()}:${port}`,
```

Comment:
Should baseUrl be documented in the options above?

Reply:
baseUrl is specific to correctly configuring webdriver.io, not necessarily this service specifically. It is described under the Wdio_Utility configuration as follows:

> Finally, the `baseUrl` must be specified to be the site to be tested. Use the [ip](https://www.npmjs.com/package/ip) npm package to obtain the IP address for the SeleniumDocker instance.

It would be helpful to reiterate that

Reply:
Ok sounds good to me